src/Essentials/src/Battery/Battery.android.cs

```diff
@@ -1,3 +1,5 @@
+# nullable enable
```

Comment:
I normally see it as:
```suggestion
#nullable enable
```

But I guess it works either way! The code formatter will probably fix it later, so no need to change it.